deployment/README.md

```diff
@@ -39,12 +39,20 @@ Stack launching is managed with `mmw_stack.py`. This command provides an interfa
 
 Before launching the Model My Watershed stack, AMIs for each service need to be generated:
 
-
 ```bash
 $ ./mmw_stack.py create-ami --aws-profile mmw-stg --mmw-profile staging \
                             --machine-type mmw-{app,monitoring,tiler,worker}
 ```
 
+### Pruning AMIs
+
+After creating several AMIs, older ones become stale and are no longer needed. To prune them use:
+
+```bash
+$ ./mmw_stack.py prune-ami --aws-profile mmw-stg --mmw-profile staging \
+                           --keep 5 --service {Application,Monitoring,Tiler,Worker}
```

Comment:
You're right. Addressed in https://github.com/WikiWatershed/model-my-watershed/commit/4253e5264045890a03ed514a12a4310c6fd07518.